OpenMDAO/Aviary
# Add aviary metadata to the dashboard and add ability to copy cell values

### Summary

This PR adds showing Aviary variables metadata as a hover over popup in the Aviary Variables Tab of the dashboard. 

It also adds Copy buttons to cells so the user can easily copy the values and paste them into Python code. 

Finally, it adds a checkbox that lets the user only show Aviary Variables in the table. ( Variables whose promoted names begin with "mission:" or "aircraft".)

The easiest way to see and test this is:

```
python ./aviary/examples/external_subsystems/simple_weight/run_simple_weight.py
aviary dashboard run_simple_weight
```

Then look at the Results-Aviary Variables Tab.

Test:
1. Hover over first 3 columns to see hover over popup
2. Try the Show Only Aviary Variables checkbox
3. Try clicking on the Copy buttons to see if they copy the correct values in the correct format for use in Python code


### Related Issues

- Resolves #113 

### Backwards incompatibilities

None

### New Dependencies

None
